Makefile

```diff
@@ -2285,28 +2278,6 @@ install-certs: etc/roots.pem
 	$(Q) $(INSTALL) -d $(prefix)/share/grpc
 	$(Q) $(INSTALL) etc/roots.pem $(prefix)/share/grpc/roots.pem
 
-verify-install:
```

Comment:
You removed the "verify-install" target, but forgot to remove the dependency on that target later.